### PR TITLE
DBZ-2231 Fix test failure - SqlServerChangeTableSetIT#readHistoryAfterRestart

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-2231

### CODE REVIEW

File: debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java [hunk 1]
```diff
@@ -329,6 +329,33 @@ public static void waitForSnapshotToBeCompleted() throws InterruptedException {
         }
     }
 
+    public static void waitForStreamingRunning() throws InterruptedException {
+        int waitForSeconds = 60;
+        final MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
+        final Metronome metronome = Metronome.sleeper(Duration.ofSeconds(1), Clock.system());
+
+        while (true) {
```
Comment (gunnarmorling) [inline debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java:337]: Did you consider to use Awaitility? Would be more in line with what we do elsewhere in similar cases.

Comment (Naros) [reply debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java:337]: I did not, but I can certainly modify the PR to use it instead.  I'll push an update shortly.

**Comment by Naros** [comment]
The test failure is unrelated but logged as https://issues.redhat.com/browse/DBZ-2319

**Comment by jpechane** [comment]
@Naros Applied, thanks!

**Pull request merged**